test/client/actions/admin_campaigns.test.js

```diff
@@ -4,7 +4,7 @@ import { destroy } from 'redux-form';
 
```

Comment:
Jadzia and I had a conversation in which she communicated that she would off on testing whether the is_admin prop is rendering the proper component in favor of moving on to more urgent tickets this sprint. This is a comment to note that we should add a ticket to the backlog for adding is_admin tests.

Reply:
I also added a comment above to add some docs for the common components maybe that can be part of that ticket too/ an additional ticket?